app/vmbackup/README.md

```diff
@@ -167,6 +167,23 @@ See [this article](https://medium.com/@valyala/speeding-up-backups-for-big-time-
   -customS3Endpoint=https://s3-fips.us-gov-west-1.amazonaws.com
 ```
 
+* Usage with official [docker-compose](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/deployment/docker/docker-compose.yml) with  `vmsingle` instance:
+```
+  vmbackup:
+    image: victoriametrics/vmbackup:latest
+    volumes:
+      - vmdata:/storage
+      - /data:/data
+    command:
+      - '--storageDataPath=/storage'
+      - '--snapshot.createURL=http://victoriametrics:8428/snapshot/create'
+      - '--snapshotName=your_snapshot_name_here'
```

Comment:
The `--snapshotName` arg isn't needed if `--snapshot.createURL` arg is set. Made docs more clear about this at 79ec35cef9310e342cebc30c7d4e50443ffc64cb